packages/platform-sdk-xrp/src/services/message.ts

```diff
@@ -13,7 +13,7 @@ export class MessageService implements Contracts.MessageService {
 
 	public async sign(input: Contracts.MessageInput): Promise<Contracts.SignedMessage> {
 		try {
-			const { publicKey, privateKey } = deriveKeypair(BIP39.normalize(input.mnemonic));
+			const { publicKey, privateKey } = deriveKeypair(BIP39.normalize(input.mnemonic!));
```

Comment:
This needs a proper check that the mnemonic is set now that it isn't enforced by TypeScript.